app/controllers/api/v1/users_controller.rb

```diff
@@ -50,6 +101,17 @@ def by_location
 
       private
 
+      # set_social_user
```

Comment:
I failed to mention this, but typically only `public` methods are documented.  Unless they are confusing, or justify documentation for future developer understanding.  

So you can remove the docs on this one.

Reply:
done.